src/include/uapi/ipc/stream.h

```diff
@@ -111,13 +117,16 @@ struct sof_ipc_stream_params {
 	/* for notifying host period has completed - 0 means no period IRQ */
 	uint32_t host_period_bytes;
 
+	uint32_t reserved[2];
```

Comment:
@lgirdwood  could reserved[2] be padded at the end of structure ?  so that it doesn't make a IPC data invalidate if it is reused for other purpose. And 2 is enough?